server/build.gradle.kts

```diff
@@ -114,6 +114,6 @@ tasks {
     environment("GRAVITINO_TEST", "true")
   }
   clean {
-    delete("$propertiesFile")
+    delete(file(propertiesFile).getParentFile())
```

Comment:
So, are you going to remove the directory `resources`? I don't believe we need to take any action here. as 
line
```
delete("$propertiesFile")
```

has done the work.

Reply:
since the directory is generated by gradle build
https://github.com/datastrato/gravitino/blob/83e0e6b7a0b9dea689c90e34416a195385bbe835/server/build.gradle.kts#L87
currently, gradle clean remove the file inside and leave the empty directory, which is untracked by git.